cabby/geo/walk.py

```diff
@@ -130,6 +144,7 @@ def get_start_poi(map: map_structure.Map, end_point: GeoDataFrame) -> \
 
     return start_point
 
+
 def get_landmark_if_tag_exists(gdf: GeoDataFrame, tag: Text, main_tag:
                                Text, alt_main_tag: Text) -> GeoDataFrame:
```

Comment:
Don't break line at type annotation. (Go ahead and put each arg on its own line if necessary.)